docs/configuration.md

```diff
@@ -121,7 +121,7 @@ app:
     path: path/to/private-key.pem
 
 datafeed:
-  version: v1
+  version: v2
```

Comment:
Please update line 191 saying the default is DF v2.

Reply:
I forgot that, thank you, done!